### PR TITLE
Hardcode plugin location to EPG menu

### PR DESCRIPTION
**Pull request by AbuBaniaz**
Can be toggled in plugin menu. Default = True (for now)
EPG menu is not EPG settings

### CODE REVIEW

File: src/EPGImport/plugin.py [hunk 1]
```diff
@@ -1065,19 +1080,20 @@ def Plugins(**kwargs):
 			wakeupfnc = getNextWakeup
 		),
 		PluginDescriptor(
-			name= _("EPGImport"),
+			name= "EPG importer",
```
Comment (WanWizard) [inline src/EPGImport/plugin.py]: Shouldn't this be translatable?

File: src/EPGImport/plugin.py [hunk 2]
```diff
@@ -1065,19 +1080,20 @@ def Plugins(**kwargs):
...
 			description = description,
-			where = PluginDescriptor.WHERE_PLUGINMENU,
-			icon = 'plugin.png',
-			fnc = main
+			where = PluginDescriptor.WHERE_MENU,
+			fnc = main_menu
 		),
 		PluginDescriptor(
 			name= "EPG importer",
```
Comment (WanWizard) [inline src/EPGImport/plugin.py:1089]: And this?

**Review by WanWizard — CHANGES_REQUESTED** [review]

File: src/EPGImport/plugin.py (hunk 1 above)
Comment (AbuBaniaz) [reply src/EPGImport/plugin.py]: This is part of the execute in main menu code. It isn't used. Wording used is actually on line 731.

I will nonetheless submit the change to progress the matter.

**Review by WanWizard — APPROVED** [review]

**Comment by WanWizard** [comment]
Sorry for the mess, struggling a bit with this review thing. 

What I meant to say: the text that ends up in the menu should be translatable, for other strings it isn't relevant.

**Comment by AbuBaniaz** [comment]
That text was on line 731. I had not changed that initially. I have now changed it to `EPG Importer now`. I hope there will be no uproar by adding `now`

It is still translatable.

EPG importer now

**Comment by AbuBaniaz** [comment]
I have found  crash. Will re-submit later

**Pull request closed without merging**